LibreNMS/Validations/Poller/CheckPythonWrapper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * CheckPythonWrapper.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * @link       https://www.librenms.org
+ *
+ * @copyright  2022 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
+ */
+
+namespace LibreNMS\Validations\Poller;
+
+use App\Models\Poller;
+use LibreNMS\ValidationResult;
+
+class CheckPythonWrapper implements \LibreNMS\Interfaces\Validation
+{
+    /** @var bool */
+    private $could_not_check_cron = false;
+
+    /**
+     * @inheritDoc
+     */
+    public function validate(): ValidationResult
+    {
+        if (Poller::exists() || $this->wrapperCronEnabled()) {
+            return $this->checkPythonWrapper();
+        }
+
+        if ($this->could_not_check_cron) {
+            return ValidationResult::warn(trans('validation.validations.poller.CheckPythonWrapper.cron_unread'));
```

Comment:
Hm i think we should add another level. Notice or something. Because it's not really a warning if we can't read those folders, quite the opposite I'd say, it's a good secuity measure :)

Reply:
ok is basically the same as info/notice.

Reply:
Turns out info already exists :D

Reply:
it always shows on my system, because I don't have a cron installed.